### ISSUE TITLE
gvisor commit cbd8628 missing from repository

### ISSUE THREAD

**Issue by paralin** [google/gvisor#11085]
### Description

Downstream repositories are depending on a commit that is now missing in this repository.

To fix this, please push commit cbd86285d259 to this repository (tag it).

### Steps to reproduce

The downstream repo is netbird:

```
git clone -b v0.29.1 https://github.com/netbirdio/netbird
cd netbird
export GOPROXY=direct
go mod tidy
```

go: gvisor.dev/gvisor@v0.0.0-20230927004350-cbd86285d259: invalid version: unknown revision cbd86285d259

**Comment by milantracy** [comment]
is gvisor needed by netbird@v0.29.1?

```sh
~/workspace/netbird (tags/v0.29.1) $ go mod why gvisor.dev/gvisor
# gvisor.dev/gvisor
(main module does not need package gvisor.dev/gvisor)
```

**Comment by paralin** [comment]
Regardless the go.mod has it and the module download fails. (For context, this issue came up while working on a buildroot distro package for netbird)

**Comment by milantracy** [comment]
i think that release got some cherrypick commits, you can work it around by doing

```sh
~/workspace/netbird (tags/v0.29.1) $ go get gvisor.dev/gvisor@release-20231003.0
go: downloading gvisor.dev/gvisor v0.0.0-20230927235549-0e18a826d91d
go: upgraded gvisor.dev/gvisor v0.0.0-20230927004350-cbd86285d259 => v0.0.0-20230927235549-0e18a826d91d
~/workspace/netbird (tags/v0.29.1*) $ git --no-pager diff
diff --git a/go.mod b/go.mod
index 16b5b55e..c7e5f84a 100644
--- a/go.mod
+++ b/go.mod
@@ -228,7 +228,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637 // indirect
-	gvisor.dev/gvisor v0.0.0-20230927004350-cbd86285d259 // indirect
+	gvisor.dev/gvisor v0.0.0-20230927235549-0e18a826d91d // indirect
 	k8s.io/apimachinery v0.26.2 // indirect
 )
 
diff --git a/go.sum b/go.sum
index 2355f6f0..8b8079f4 100644
--- a/go.sum
+++ b/go.sum
@@ -1234,6 +1234,8 @@ gotest.tools/v3 v3.5.0 h1:Ljk6PdHdOhAb5aDMWXjDLMMhph+BpztA4v1QdqEW2eY=
 gotest.tools/v3 v3.5.0/go.mod h1:isy3WKz7GK6uNw/sbHzfKBLvlvXwUyV06n6brMxxopU=
 gvisor.dev/gvisor v0.0.0-20230927004350-cbd86285d259 h1:TbRPT0HtzFP3Cno1zZo7yPzEEnfu8EjLfl6IU9VfqkQ=
 gvisor.dev/gvisor v0.0.0-20230927004350-cbd86285d259/go.mod h1:AVgIgHMwK63XvmAzWG9vLQ41YnVHN0du0tEC46fI7yY=
+gvisor.dev/gvisor v0.0.0-20230927235549-0e18a826d91d h1:QtuLKcvzNfIKLmzqA2FHH7iE1uyi7eqg6OA7Yswb27c=
+gvisor.dev/gvisor v0.0.0-20230927235549-0e18a826d91d/go.mod h1:AVgIgHMwK63XvmAzWG9vLQ41YnVHN0du0tEC46fI7yY=
 honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
```


let me know how it goes

**Comment by nlacasse** [comment]
That commit was removed because it contained very large pre-built binaries that significantly slowed repo clone times, so we'd rather not put it back (and I'm not even sure if we could at this point). 

Changing netbird's dependency as @milantracy suggested would be the preferred approach.

**Comment by paralin** [comment]
Thanks, I see @milantracy  submitted a PR to netbird at netbirdio/netbird#2789

**Comment by ayushr2** [comment]
Just to add some more context to what @nlacasse said, there was a period where we were committing those binaries (namely `runsc/cmd/metricserver/metricserver.flate` and `runsc/cmd/metricserver/metricserver.bin`) into the `go` branch by mistake. We fixed our release process to not commit these binaries. But during that period, the size of our .git repository increased to 1.5 GiB as we were accumulating all these binaries in the git history. Clone times were impacted (it took me 30 seconds to clone the repo).

So we used `git-filter-repo(1)` + `git gc` to remove these binaries from the go branch history and it reduced our repo size to 115 MiB. Hence the commit is missing now.
